README.md

```diff
@@ -1,48 +1,129 @@
-https://github.com/CommonGateway/ShopBundle (voorheen product, order, invoice en payment componenten)
-https://github.com/CommonGateway/CommunicationBundle (voorheen e-mail en sms componenten)
-https://github.com/CommonGateway/TemplateBundle (voorheen template component)
+1.  **CoreBundle**: [GitHub Repository](https://github.com/CommonGateway/CoreBundle)
+2.  **BRPBundle**: [GitHub Repository](https://github.com/CommonGateway/BRPBundle)
+3.  **KlantenBundle**: [GitHub Repository](https://github.com/CommonGateway/KlantenBundle)
+4.  **ZGWBundle**: [GitHub Repository](https://github.com/CommonGateway/ZGWBundle)
 
-Al deze bundles kunnen nog steeds als standalone component worden geïnstalleerd (zie daarvoor de individuele installatie handleidingen), maar worden vanuit de huwelijksplanner standaard als extra plugins op dezelfde gateway geïnstalleerd.
+While all these bundles can still be installed as standalone components (please refer to their respective installation guides), the HuwelijksplannerBundle now defaults to installing these bundles as additional plugins on the same gateway.
 
-\#Installeren van de huwelijksplanner
+This new approach offers greater modularity, making it easier for developers to work with the HuwelijksplannerBundle and allowing for more flexible configurations based on the needs of individual municipalities.
 
-## Backend
+If you have any questions, suggestions, or new ideas, please don't hesitate to share them in the project repository. Let's work together to create an exceptional wedding planning experience that brings joy and happiness to couples preparing for their special day.
 
-Voor de backend installatie geld dat de Common Gateway installatiehandleiding gevolgd kan worden (als de gemeente nog geen Common Gateway geïnstalleerd heeft) de handleiding treft u [hier](https://github.com/ConductionNL/commonground-gateway#readme). Voor de opzet van de backend maakt het niet uit hoe u de gateway installeert (bijvoorbeeld Haven, Kubernetes, Linux of Azure) of welke database optie u kiest (MySQL, PostgreSQl, Oracle, MsSQL). Het gateway framework handelt deze abstractie af.
+## Backend Installation Instructions
 
-Na het installeren van de Gateway logt u in en vindt u onder “plugins” in het linker menu een overzicht van de reeds geïnstalleerde plugins. Als de huwelijksplanner hier nog niet tussen staat kunt u rechtboven in op “Search” klikken en op “Huwelijksplanner” zoeken. Klik op de Card van huwelijksplanner en vervolgens op de knop installeren.
+The Utrecht Huwelijksplanner backend codebase utilizes the Common Gateway as an open-source installation framework. This means that the Huwelijksplanner library, in its core form, functions as a plugin on this Framework. To learn more about the Common Gateway, you can refer to the documentation [here](https://commongateway.readthedocs.io/en/latest/).
 
-## Frontend
+Please note that the Huwelijksplanner frontend codebase is a separate Kubernetes container.
```

Comment:
I'd opt for 'docker container' instead of 'kubernetes container' here. If you really want to say something about kubernetes you could add 'which results in a separate deployment in a Kubernetes cluster'.